synapse/storage/databases/main/cache.py

```diff
@@ -52,6 +53,21 @@
 # As above, but for invalidating room caches on room deletion
 DELETE_ROOM_CACHE_NAME = "dr_cache_fake"
 
+# How long between cache invalidation table cleanups, once we have caught up
+# with the backlog.
+REGULAR_CLEANUP_INTERVAL_MILLISEC = 60 * 60 * 1000
```

Comment:
Things would be a little more clear if we composed things with some labels like the following:

```py
ONE_HOUR_MS = 60 * 60 * 1000
ONE_DAY_MS = ONE_HOUR_MS * 24
```

But I think we can go one step better and just use `REGULAR_CLEANUP_INTERVAL_MILLISEC = Config.parse_duration("1h")` directly which seems a lot more clear.

Reply:
that would be a fairly nice way to write it indeed, didn't think of using the config machinery